### ISSUE TITLE
[vmbackup] Error while creating backup

### ISSUE THREAD

**Issue by hollden** [VictoriaMetrics/VictoriaMetrics#856]
**Describe the bug**
When I start to create backup from instant snapshot, it ends with the error:
```
2020-10-23T08:10:18.462Z        info    VictoriaMetrics/lib/backup/actions/backup.go:171        backup from src fslocal "/vmetrics/vmstorage//snapshots/20201023080930-164042F08AA068D4" to dst fsremote "/vmetrics/vmstorage/backup" with origin fsnil is complete; backed up 2183124983 bytes in 8.766 seconds; deleted 0 bytes; server-side copied 0 bytes; uploaded 2183124983 bytes
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x50 pc=0xbd72b7]

goroutine 1 [running]:
main.main()
        github.com/VictoriaMetrics/VictoriaMetrics/app/vmbackup/main.go:95 +0x3f7
```
So I consider that backup ends sucsessfully, but this error is bothering me. Maybe I doing something wrong, or it is a bug, or I should just don't pay attention on this error?

**To Reproduce**
Command which I try to make backup:
`vmbackup-prod -storageDataPath=/vmetrics/vmstorage/ -snapshotName=20201023080930-164042F08AA068D4 -dst=fs:///vmetrics/vmstorage/backup`

**Version**
The line returned when passing `--version` command line flag to binary. For example:
```
vmbackup-prod --version
vmbackup-20201013-140114-tags-v1.44.0-0-g94978af9b
```

**Comment by f41gh7** [comment]
seems like omitted ``--origin`` flag leads to panic.  @kreedom  PTAL

**Comment by f41gh7** [comment]
Commit https://github.com/VictoriaMetrics/VictoriaMetrics/commit/a51a7b2a206aaf42848b1be9cc1fdf837de272a6 fixes this bug, it will be included at the nearest release.

**Comment by hollden** [comment]
Thank you!

**Comment by valyala** [comment]
FYI, the bugfix has been included in [v1.45.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases).
